apps/bandwagon/views.py

```diff
@@ -370,12 +370,15 @@ def ajax_list(request):
         return http.HttpResponseBadRequest()
 
     # Get collections associated with this user
-    collections = Collection.objects.publishable_by(request.amo_user)
-
-    for collection in collections:
-        # See if the collections contains the addon
-        if addon_id in collection.addons.values_list('id', flat=True):
-            collection.has_addon = True
+    has_addon = """
+        select 1 from addons_collections as ac
```

Comment:
This `select 1` only works in this case because in python `True == 1` (but for example `True == 2` is False). I'm talking specifically about the use of `has_addon` here: https://github.com/mozilla/olympia/blob/master/apps/bandwagon/templates/bandwagon/ajax_list.html#L4

I'm thus a bit worried about this. Maybe we could write a version of `class_selected` which is a bit more lax and only tests for truthiness? Or write it the dumb way `{% if collection.has_addon %}class="selected"{% endif %}`?

The very minimum would be to make a note in a comment (here and in the template?) explaining why this works.